### ISSUE TITLE
Outline

### ISSUE THREAD

**Issue by landac** [landac/PlantSeedlingClassificationCNN#1]
# Outline

##User Story
As an end user, I want an image (Convolutional Neural Network) classifying algorithm ready for productionizing, 
so that I can identify crop seedling species at an early stage of the plant's lifecycle.

## Acceptance Criteria
It is done when the best trained model's validation set performance metrics are above the threshold selected after Exploratory Data Analysis is performed, leveraging noise, bias and computational cost.
